acq4/devices/LEDLightSource/LEDLightSource.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.Device import *
+from acq4.devices.LightSource import *
+from acq4.devices.DAQGeneric import DAQGeneric, DAQGenericTaskGui
+from PyQt4 import QtCore, QtGui
+import acq4.util.Mutex as Mutex
+
+class LEDLightSource(LightSource):
+    """Simple device which reports the status of the LED Light Sources...reports up to the LightSource object."""
+    
+    def __init__(self, dm, config, name):
+        LightSource.__init__(self, dm, config, name)
+        self.ledconfig = config.get('leds', config)
```

Comment:
Ok...I was making the assumption (from looking at other examples), that the second "config" was required in the config.get call.